integrations/satismeter/lib/index.js

```diff
@@ -54,6 +63,7 @@ SatisMeter.prototype.identify = function(identify) {
   window.satismeter({
     writeKey: this.options.apiKey || this.options.token,
     userId: identify.userId(),
+    anonymousId: identify.anonymousId(),
     traits: this.analytics.user().traits(),
     type: 'identify'
```

Comment:
@E-Mud I think we should convert this to the new format as well.

We already expect that the `load` method exists, so we can safely expect that `identify` method exists as well.

Reply:
Done -> e1157ef